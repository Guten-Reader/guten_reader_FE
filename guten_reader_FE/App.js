import {createAppContainer} from 'react-navigation';
import {createStackNavigator} from 'react-navigation-stack';
import Reader from './src/Components/Reader/Reader.js';
import Library from './src/Components/Library/Library.js';
import Search from './src/Components/Search/Search.js';
import 'react-native-gesture-handler';

console.disableYellowBox = true;

const MainNavigator = createStackNavigator({

  Library: {
    screen: Library,
    navigationOptions: {
      header: null,
    }
  },
  Reader: {
    screen: Reader,
  },
  Search: {
    screen: Search,
  }
});

const App = createAppContainer(MainNavigator);

export default App;
