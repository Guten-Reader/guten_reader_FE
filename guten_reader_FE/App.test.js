import React from 'react';
// import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import App from './App';

describe('App renders correctly with Jest', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  })
})