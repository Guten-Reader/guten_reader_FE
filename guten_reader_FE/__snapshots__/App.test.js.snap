// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders correctly with Jest renders correctly 1`] = `
<ContextProvider
  value="light"
>
  <ContextProvider
    value={
      Object {
        "_childrenNavigation": Object {},
        "actions": Object {
          "dismiss": [Function],
          "pop": [Function],
          "popToTop": [Function],
          "push": [Function],
          "replace": [Function],
          "reset": [Function],
        },
        "addListener": [Function],
        "dangerouslyGetParent": [Function],
        "dismiss": [Function],
        "dispatch": [Function],
        "getChildNavigation": [Function],
        "getScreenProps": [Function],
        "goBack": [Function],
        "isFirstRouteInParent": [Function],
        "isFocused": [Function],
        "navigate": [Function],
        "pop": [Function],
        "popToTop": [Function],
        "push": [Function],
        "replace": [Function],
        "reset": [Function],
        "router": Object {
          "childRouters": Object {
            "Library": null,
            "Reader": null,
            "Search": null,
          },
          "getActionCreators": [Function],
          "getActionForPathAndParams": [Function],
          "getComponentForRouteName": [Function],
          "getComponentForState": [Function],
          "getPathAndParamsForState": [Function],
          "getScreenOptions": [Function],
          "getStateForAction": [Function],
        },
        "setParams": [Function],
        "state": Object {
          "index": 0,
          "isTransitioning": false,
          "key": "StackRouterRoot",
          "routes": Array [
            Object {
              "key": "id-1578444446219-0",
              "routeName": "Library",
            },
          ],
        },
      }
    }
  >
    <KeyboardAwareNavigator
      navigation={
        Object {
          "_childrenNavigation": Object {},
          "actions": Object {
            "dismiss": [Function],
            "pop": [Function],
            "popToTop": [Function],
            "push": [Function],
            "replace": [Function],
            "reset": [Function],
          },
          "addListener": [Function],
          "dangerouslyGetParent": [Function],
          "dismiss": [Function],
          "dispatch": [Function],
          "getChildNavigation": [Function],
          "getScreenProps": [Function],
          "goBack": [Function],
          "isFirstRouteInParent": [Function],
          "isFocused": [Function],
          "navigate": [Function],
          "pop": [Function],
          "popToTop": [Function],
          "push": [Function],
          "replace": [Function],
          "reset": [Function],
          "router": Object {
            "childRouters": Object {
              "Library": null,
              "Reader": null,
              "Search": null,
            },
            "getActionCreators": [Function],
            "getActionForPathAndParams": [Function],
            "getComponentForRouteName": [Function],
            "getComponentForState": [Function],
            "getPathAndParamsForState": [Function],
            "getScreenOptions": [Function],
            "getStateForAction": [Function],
          },
          "setParams": [Function],
          "state": Object {
            "index": 0,
            "isTransitioning": false,
            "key": "StackRouterRoot",
            "routes": Array [
              Object {
                "key": "id-1578444446219-0",
                "routeName": "Library",
              },
            ],
          },
        }
      }
      theme="light"
    />
  </ContextProvider>
</ContextProvider>
`;
